packages/voyager-keycloak/src/KeycloakSecurityService.ts

```diff
@@ -97,6 +97,9 @@ export class KeycloakSecurityService implements SecurityService {
   }
 
   public async onSubscriptionConnect(connectionParams: any, webSocket: any, context: any): Promise<any> {
+    if (!connectionParams || typeof connectionParams !== 'object') {
```

Comment:
Second check covers first as well.
```suggestion
    if (typeof connectionParams !== 'object') {
```